tuning/src/main/java/frc/robot/subsystems/IntakeTuningSubsystem.java

```diff
@@ -0,0 +1,112 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkBase.ControlType;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class IntakeTuningSubsystem extends SubsystemBase {
```

Comment:
I would not use a command based robot for this. I would create the subsystems in robot.java and just manually manage them there. You can get rid of the subystembase inheritance, robotcontainer and anything else command based related.